spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  let(:my_user)   { create(:user) }
```

Comment:
Consider modifying the user factory so it creates confirmed users by default as confirmed users are usually desired for tests. This would allow line 8 `my_user.confirm` to be removed.

Here's an example of a user factory that creates confirmed users: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/factories/users.rb